### ISSUE TITLE
Use of non alphabetic characters for MarkLogic password makes creation of model impossible

### ISSUE THREAD

**Issue by peetkes** [Smartlogic-Semaphore-Limited/docker-examples#2]
I used some non-alphabetic characters in my MARKLOGIC_ADMIN_PASSWORD which caused the creation of a new model to fail because it could not connect to the marklogic backend.
When I changed my MARKLOGIC_ADMIN_PASSWORD to all alphabetic characters I was able to create a new model

**Comment by TheKeithAtkins** [comment]
Non-alphanumeric characters in the MARKLOGIC_ADMIN_PASSWORD will need to be URL encoded. For instance, if the password is admin!, then this should be presented as admin%21
